demos/demo3-todomvc-list-transition/Demo.jsx

```diff
@@ -133,9 +133,9 @@ const Demo = React.createClass({
           <h1>todos</h1>
           <form onSubmit={this.handleSubmit}>
             <input
+              autoFocus
```

Comment:
Eh, Facebook convention is to have the attribute value always there. I'd like to abide by that.